frontends/election-manager/src/components/smartcard_modal/program_election_card_view.tsx

```diff
@@ -61,31 +75,42 @@ export function ProgramElectionCardView({
   }
 
   return (
-    <Prose textCenter>
-      <h2>Program Election Card</h2>
-      {/* An empty div to maintain space between the header and subsequent p. TODO: Consider adding
-        a `maintainSpaceBelowHeaders` prop to `Prose` */}
-      <div />
-      {actionStatus && <StatusMessage actionStatus={actionStatus} />}
+    <Prose textCenter theme={fontSizeTheme.medium}>
+      {actionStatus && (
+        <p>
+          <StatusMessage actionStatus={actionStatus} />
+        </p>
+      )}
+      <Heading marginTop={showingSuccessOrErrorMessage ? '1em' : '0'}>
+        Create New Election Card
+      </Heading>
+      {electionDefinition && (
+        <p>{electionToDisplayString(electionDefinition.election)}</p>
+      )}
+
+      <HorizontalRule />
       {electionDefinition ? (
-        <p>Admin and Poll Worker cards only work for the current election.</p>
+        <p>
+          <Button disabled={!electionDefinition} onPress={programAdminCard}>
+            Admin Card
+          </Button>{' '}
+          or{' '}
```

Comment:
The "or" was a bit jarring when I saw this screen. I feel like having two buttons next to each other is an implicit "or" already and we don't need to actually write the word

Reply:
@jonahkagan - Yeah, this is less common in our app. And eventually you won't need to make this choice once we know what type of card this is.

Reply:
So just to double check @beausmith, we still want to keep the "or" here, then?